poc/vdaf_pine.py

```diff
@@ -308,8 +309,9 @@ def prep_init(Pine,
         # compute the dot products in wraparound checks.
         wr_joint_rand_xof = \
             Pine.wr_joint_rand_xof(k_corrected_wr_joint_rand_seed)
-        wr_dot_prods = Pine.Flp.Valid.run_wr_checks(meas_share,
-                                                    wr_joint_rand_xof)
+        wr_dot_prods = Pine.Flp.Valid.run_wr_checks(
```

Comment:
While we're here, should this be `wr_check_results`?
```suggestion
        wr_check_results = Pine.Flp.Valid.run_wr_checks(
```